scripts/init.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 brew install swift-format ktlint release-it
 yarn
-yarn build
+yarn prepare
```

Comment:
Are you sure that his is expected change? with `build` we wanted TS files to be generated for local usage + release. `prepare` should do other stuff 🤔

Reply:
The thing was that `expo-module build` doesn't build the plugin on its own, causing the example not to work. Additionally, it runs with `--watch` automatically so there is a need for `EXPO_NONINTERACTIVE=1` before every command. 

When you look at the [source code of prepare](https://github.com/expo/expo/blob/main/packages/expo-module-scripts/bin/expo-module-prepare) ` it just runs `expo-module build` with different params to build everything. 

Anyway, I reverted to the previous and updated `package.json` to run only the builds that we needed.